config/config.go

```diff
@@ -59,6 +59,7 @@ func InitFlags() {
 	// TLS is not currently supported
 
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	viper.BindPFlags(pflag.CommandLine)
```

Comment:
As we're doing this in ParseCommandLine() should it be duplicated now?

Reply:
To Test if the values are initialised correctly, the present configuration is empty as the viper flags are not updated. so edited that function, if the test seems irrelevant, we can avoid the duplication.